### ISSUE TITLE
JUnitXmlPrinter$$escape breaks with NullPointerException when ran in SBT

### ISSUE THREAD

**Issue by SemanticBeeng** [etorreborre/specs2#344]
### Error

```
[error] java.lang.NullPointerException
[error] 
[error] STACKTRACE
[error]  scala.xml.Utility$.escape(Utility.scala:118)
[error]  scala.xml.Utility$$anonfun$escape$1.apply(Utility.scala:89)
[error]  scala.xml.Utility$$anonfun$escape$1.apply(Utility.scala:89)
[error]  scala.xml.Utility$.sbToString(Utility.scala:33)
[error]  scala.xml.Utility$.escape(Utility.scala:89)
[error]  org.specs2.reporter.JUnitXmlPrinter$class.org$specs2$reporter$JUnitXmlPrinter$$escape(JUnitXmlPrinter.scala:114)
[error]  org.specs2.reporter.JUnitXmlPrinter$TestCase.xml(JUnitXmlPrinter.scala:88)
[error]  org.specs2.reporter.JUnitXmlPrinter$TestSuite$$anonfun$xml$3.apply(JUnitXmlPrinter.scala:71)
[error]  org.specs2.reporter.JUnitXmlPrinter$TestSuite$$anonfun$xml$3.apply(JUnitXmlPrinter.scala:71)
[error]  scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:245)
[error]  scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:245)
```
### Test case

```scala
// Specs2

import org.scalatest.Retries
import org.specs2._
import org.specs2.execute.Success
import org.specs2.matcher.{FutureMatchers, ResultMatchers}

import scala.concurrent.{ExecutionContext, Future}
import scala.language.{higherKinds, postfixOps}

//JUnit

import org.junit.runner.RunWith
import org.specs2.runner.JUnitRunner

import scala.concurrent.duration._

/**
 *
 */
@RunWith(classOf[JUnitRunner])
class LoansServerApi_CaptureSpec extends mutable.Specification with ResultMatchers with FutureMatchers with Retries {

  override def is =
    section("travis") ^
      sequential ^
      s2"""


     End everything well
     ${ implicit ec: EC => {

        Future {
          "Really done".pp
          Success("done")
        }

      } await(retries = 0, timeout = 100.seconds)
      }
    """

  type EC = ExecutionContext
}
```

**Comment by SemanticBeeng** [comment]
No, actually this code works fine in scpecs2 project.
Breaks like that only in mine... Why would this be?

**Comment by etorreborre** [comment]
From reading the stacktrace I am not really sure what's going on here. I have added, in `3.1-20150321223007-81e756a`, some additional null checks which might help. Please tell me if that works better for you.

**Comment by etorreborre** [comment]
Please try my fix in `3.1.1` and reopen if that doesn't work.
